InitializeRepo/readme.md

```diff
@@ -18,19 +18,59 @@ Make sure you have the following prerequisites installed:
 1. Open a terminal or command prompt.
 
 2. Make the script executable if it isn't already by running the following command:
-    ```bash
+    ```
```

Comment:
You can keep bash here since it’s a single command, though it’s not a big difference.